CMSIS/NN/Source/ConvolutionFunctions/arm_convolve_1x1_HWC_q7_fast_nonsquare.c

```diff
@@ -157,7 +157,7 @@ arm_status arm_convolve_1x1_HWC_q7_fast_nonsquare(const q7_t * Im_in,
                 q31_t     inA1, inA2;
                 q31_t     inB1, inB2;
 
-                pA = (const q7_t *)read_and_pad_reordered((void *)pA, &inA1, &inA2);
+                pA = read_and_pad_reordered(pA, &inA1, &inA2);
 
                 inB1 = *__SIMD32(pB)++;
```

Comment:
Since this commit is introducing arm_nn_read_q7x4_ia at some places to replace SIMD32 then I think the few remaining SIMD32 should be replaced too.